wapitiCore/language_sources/en.po

```diff
@@ -238,6 +238,10 @@ msgstr "Found backup file !"
 msgid "Backup file {0} found for {1}"
 msgstr "Backup file {0} found for {1}"
 
+#: ../vulnerability.py:79 ../language/vulnerability.py:87
+msgid "Basic Authentication Brute Force"
+msgstr "Basic Authentication Brute Force"
+
```

Comment:
Translations for "Basic Authentication Brute Force description" and "Basic Authentication Brute Force solution" are missing.

Reply:
fixed